apiserver/facades/controller/crossmodelrelations/crossmodelrelations.go

```diff
@@ -148,7 +148,8 @@ func (api *CrossModelRelationsAPIv3) publishOneRelationChange(ctx context.Contex
 	case change.Life != life.Alive:
 		// Relations only transition to dying and are removed, so we can safely
 		// just remove the relation and return early.
-		return api.removalService.RemoveRemoteRelation(ctx, relationUUID)
+		_, err := api.removalService.RemoveRemoteRelation(ctx, relationUUID, false, 0)
+		return err
```

Comment:
```suggestion
		_, err := api.removalService.RemoveRemoteRelation(ctx, relationUUID, true, time.Minute)
		return err
```

Use force, we want to see it gone and give it at least a minute before going full brute force.